docs/source/data/kedro_dvc_versioning.md

```diff
@@ -59,17 +69,30 @@ data/**
 !data/**/
 ```
 
-The metadata file can then be commited to git.
+Once updated, add the `.dvc` file to Git and commit the changes:
+
+```bash
+git add data/01_raw/companies.csv.dvc
+git commit -m "Track companies.csv dataset with DVC"
```

Comment:
This `git commit` comes at a weird moment because we haven't added any other file yet.

For example, we could tell the user to do `git init` followed by `git add . && git commit -m 'First commit, initial structure from starter'`.